framework/Models/PostProcessors/validationAlgorithms/ValidationBase.py

```diff
@@ -72,10 +68,11 @@ def initialize(self, features, targets, **kwargs):
     self.pivotParameter = kwargs.get('pivotParameter')
     self.metrics = kwargs.get('metrics')
     if self.acceptableMetrics:
-      accetable = [True if metric.estimator.type in self.acceptableMetrics else False for metric in self.metrics]
-      if not all(accetable):
-        notAcceptable = [self.metrics[i].estimator.type for i, x in enumerate(accetable) if not x]
-        self.raiseAnError(IOError, "The metrics '{}' are not acceptable for validation algorithm {}".format(','.join(notAcceptable), self.name))
+      acceptable = [True if metric.estimator.isInstanceString(self.acceptableMetrics) else False for metric in self.metrics]
+      if not all(acceptable):
+        notAcceptable = [self.metrics[i].estimator.type for i, x in enumerate(acceptable) if not x]
```

Comment:
With the new changes, the ```self.metrics[i].estimator.type``` is also changed, right? The old one represents actual subType, but new one is now just "Metric", right?

Reply:
Yes. I will look at this more closely, but you will note all the tests pass this check successfully currently.

Reply:
Oh, I see, you want the print statement updated. I can figure out how to do that, probably.

Reply:
I added a PluginReadyEntity property to get the string representation of the Entity's Interface type.